packages/material-ui/src/Slide/Slide.d.ts

```diff
@@ -6,6 +6,7 @@ export interface SlideProps extends TransitionProps {
   direction: 'left' | 'right' | 'up' | 'down';
   ref?: React.Ref<unknown>;
   theme?: Theme;
+  parent?: React.Ref<HTMLDivElement | null>;
```

Comment:
We suffix our refs with `Ref`

```suggestion
  parentRef?: React.Ref<HTMLDivElement | null>;
```